roles/openshift_on_openstack/templates/prepare_block_devices.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: A playbook to prepare and mount block devices
-  hosts: openstack_master_nodes
+  hosts: masters
```

Comment:
An alternative is to use the scaleup inventory here and keep the `openstack_*_nodes` groups.

Either way should work so it's more a matter of your preference.

Reply:
I prefer to use the more generic `masters` group as it was originally before the inventory change.